### PR TITLE
{tools}[dummy/] ccache v3.2.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2062.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7b080ec122a8768c890aaa3ea07c6943 for a full test report.

File: easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description = """ccache-3.1.9: Cache for C/C++ compilers"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/ccache/ccache/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('Autotools', '20150215')]
+dependencies = [
+    ('zlib', '1.2.8'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: I would use the system one here.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2421.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3652d0f5d0a7d7a897114806e20dd960 for a full test report.

File: easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: Any particular reason? To avoid a version conflict?

Comment (wpoely86) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: yes. If we want to use `ccache` as `preload module` for EB, I would prefer that it uses 100% system deps.

Comment (boegel) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: or that we used `zlib` as a build dep and link statically... I would prefer that (strongly) over using system deps

Comment (wpoely86) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: you're already using system GCC? 

Static linking is also fine for me.

Comment (boegel) [inline easybuild/easyconfigs/c/ccache/ccache-3.2.5.eb]: Sure, but a system without GCC is a lot less likely than a system without zlib...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2646.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6cc33ed55eaac0fdce85dbe713288dd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/49373712ef951d4a49ff310b58a7f9b9 for a full test report.

**Comment by boegel** [comment]
@wpoely86 now linking `zlib` statically, thanks for the review!

**Pull request merged**